_posts/events/2022-10-04-nick-shields.md

```diff
@@ -3,29 +3,26 @@ layout: technexus_event
 categories:
   - events
 links: 
-title: "In-person and Online: Project Rainbow - An educational initiative from Cook County Board President Toni Preckwinkle"
-description: ""
+title: "In-person and Online: Project Rainbow: Address Learning Loss through a Digital Equity Lens"
+description: "There’s nothing more vital to a person’s chances of success than supporting their development in the first years of life, including easy access to high-quality early education opportunities. While nothing can replace the effectiveness of live, in-person learning, extensive research shows that children’s television programming and other forms of educational media can significantly improve children’s mental and social development. Cook County launched Project Rainbow, new high-quality educational TV programming for children and families in 2021. The content is available on County TV (Channel 22, 900 on Comcast), the Cook County website, Facebook, and on the free Cook County Project Rainbow mobile app."
 speakers:
- - "Nick Shields, Cook County Director of Communications and Public Affairs"
-image: /images/logo/logo-star-social.jpg
+ - "Nick Shields, Director of Communications and Public Affairs for Cook County & Cook County Board President Toni Preckwinkle, He, Him, His"
+image: /images/events/512-ProjectRainbowKids.jpg
 image_credit: 
 date: 2022-10-04T18:00:00-05:00
 event_id: 512
 youtube_id: 
-agenda: 
+agenda: https://docs.google.com/presentation/d/1RKxvkI5eE2mBHpROUIjs83Aeh9-DnUATEUSDPDuCADc/edit#slide=id.g121c7120608_0_0
```

Comment:
do we usually link to the slide deck here, or the google doc? (this is a link to the slide deck)

Reply:
We've started to just use the slide deck for in person events since that is what we show on the projector.

Reply:
I'm open to using though.

Reply:
Oh cool, no worries.